.github/workflows/test-asan.yml

```
@@ -30,33 +30,37 @@ concurrency:
   ASAN_OPTIONS: intercept_tls_get_addr=0
-  PYTHON_VERSION: '3.11'
+  PYTHON_VERSION: '3.12'
   FLAKY_TESTS: keep_retrying
 
 permissions:
   contents: read
 
 jobs:
   test-asan:
-    if: github.event.pull_request.draft == false
+    if: false  # Temporarily disabled. References: https://github.com/nodejs/node/pull/52374, https://github.com/nodejs/node/pull/53651#issuecomment-2198510810
```

actionlint (1.7.9)

condition "false" is always evaluated to false. remove the if: section

(if-cond)